### ISSUE TITLE
Add two necessary steps to install instruction 

### ISSUE THREAD

**Issue by jonaswjs** [LaxarJS/cube.theme#13]
- [x]  add alias to webpack configuration:
```
resolve: {
...
   alias: {
      'default.theme': 'laxar-uikit/themes/default.theme',
      'cube.theme': 'laxar-cube.theme'
   }
},
```
- [x]  install 
```
npm install --save-dev node-sass
```

**Comment by x1B** [comment]
Possibly, we should also mention the `$fa-font-path` setting.

**Comment by x1B** [comment]
Implemented on `master` (`v2.0.0`).
